cfme/infrastructure/cluster.py

```diff
@@ -111,6 +111,14 @@ def exists(self):
         except sel.NoSuchElementException:
             return False
 
+    def run_smartstate_analysis(self):
+        sel.force_navigate('infrastructure_cluster', context={
+            'cluster': self})
+        tb.select('Configuration', 'Perform SmartState Analysis', invokes_alert=True)
+        sel.handle_alert()
+        flash.assert_message_contain('Cluster / Deployment Role: scan successfully initiated'.
+            format(self.name))
```

Comment:
It is better to indent like this:
```python
flash.assert_message_contain(
    'Cluster / Deployment Role: scan successfully initiated'.format(self.name))
```

And if it gets longer than 100 chars, indent after `format(`:
```python
flash.assert_message_contain(
    'Cluster / Deployment Role: scan successfully initiated'.format(
        self.name))
```